Extension/src/LanguageServer/client.ts

```diff
@@ -3063,13 +2860,13 @@ export class DefaultClient implements Client {
             return;
         }
 
-        const isCodeAction: boolean = (adjustedLine !== undefined && adjustedLine !== undefined);
-        const initCursorPosition: vscode.Position = isCodeAction ? new vscode.Position(initCursorLine ?? 0, initCursorColumn ?? 0) : editor.selection.active;
+        const isCodeAction: boolean = codeActionArguments !== undefined;
+        const initCursorPosition: vscode.Position = codeActionArguments !== undefined ? new vscode.Position(codeActionArguments.initialCursor.line, codeActionArguments.initialCursor.character) : editor.selection.active;
```

Comment:
`codeActionArguments !=== undefined` should just be `isCodeAction`

Reply:
That's not allowed.  Evaluation of an undefined variable in a bool expression works, but not on assignment to a boolean variable.  That causes a type mismatch.